_content/booklets/kalama-sutta-free-inquiry_soma-thera.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Kālāma Sutta: The Buddha's Charter of Free Inquiry"
+authors:
+  - "Soma Thera"
+external_url: "https://www.bps.lk/olib/wh/wh008_Soma_Buddhas-Charter-of-Free-Inquiry.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1PTxGx_8w2seQv5cfYn2N7OZ4a3mIxooc/view?usp=sharing"
```

Comment:
pdf first, but the epub needs to be the second link: https://drive.google.com/file/d/1dwAY-YcrJMeoVAcBXwYI9BePhbS616fu/view?usp=drivesdk

These wheel pubs are the whole reason drive_links is plural 😅